native-audio/app/src/main/java/com/example/nativeaudio/NativeAudio.java

```diff
@@ -288,7 +305,7 @@ protected void onDestroy()
 
     /** Native methods, implemented in jni folder */
     public static native void createEngine();
-    public static native void createBufferQueueAudioPlayer();
+    public static native void createBufferQueueAudioPlayer(int rate, int size);
```

Comment:
done: change the parameter  name to reflect the usage:  sampleRate, samplesPerBuffer